firebase.json

```
@@ -66,14 +66,22 @@
           }
         ]
       }
+    ],
+    "rewrites": [
+      {
+        "source": "**",
+        "destination": "/index.html"
+      },
+      {
+        "source": "/maps",
```

this may need to be a bit more generic

```suggestion
        "source": "/maps/**",
```